test/rules/string-throws/test.js.lint

```diff
@@ -0,0 +1,23 @@
+let a = () => throw 'bla';
```

Comment:
Enabled CircleCI, added tests and logic for handling template strings, and fixed errors from 'npm run verify'